spec/integration/security_guides_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+describe 'Security Guides', swagger_doc: 'v2/openapi.json' do
+  before do
+    allow_any_instance_of(Authentication).to receive(:identity_header).and_return(
```

Comment:
Please stop using `_any_instance_of` methods, first of all it is an antipattern and second it is slowing down testing. There are plenty of [mocks](https://github.com/RedHatInsights/compliance-backend/blob/master/spec/controllers/v2/application_controller_spec.rb#L13-L14) already in existing tests that you can inspire from.